0000-settlement-engine/0000-settlement-engine.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Settlement Engine
+draft: 1
+---
+
+# Settlement Engine
```

Comment:
You could have a ledger plugin that wraps communication to the settlement engine (and I think it's definitely worth writing one). agreed that it's the successor in concept though

Reply:
> I think it would be worth mentioning somewhere near the top that this supersedes the Ledger Plugin Interface

Changed verbiage to "supercedes" here, and moved it closer to the top: https://github.com/interledger/rfcs/compare/4e742ae54dcd738b67a4a541f9070837a3c16e98...ko-settlement-engine#diff-990a915317f4e35578edfa70631bd07dR165

> You could have a ledger plugin that wraps communication to the settlement engine (and I think it's definitely worth writing one). agreed that it's the successor in concept though

I don't believe it's possible to write an SE backwards compatible with existing plugins using a wrapper -- so what's the reason for doing so?

(I think it would be nice to semi-standardize a JS interface for SEs so they can run in a browser with e.g. Rafiki, but I don't think we can use LPIv2 if we need to support the asymmetric precision as specced out here)